contessa/models.py

```diff
@@ -106,7 +107,10 @@ def init_row(
 
         self.set_medians(conn)
 
-        self.time_filter = rule.time_filter
+        if isinstance(rule.time_filter, str):
+            self.time_filter = rule.time_filter
+        else:
+            self.time_filter = json.dumps(rule.time_filter)
```

Comment:
Im not sure I get this. this is actually a bug fix, because timefilter can be a list/dict specifying multiple filters right?

Reply:
Yes, it crashed before trying to store it into TEXT